### DISCUSSION TITLE
iOS iPhone

### DISCUSSION THREAD

**Discussion by keycommander** [git-lfs/git-lfs#5545] [Q&A]
Is there a way to utilize these types of code via cellular mobile devices. Aka iPhone iOS homebrew or Macport installations. If so what alternatives are there and what names do the companies have.

**Comment by bk2204** [comment]
Hey,

I don't personally use iPhone or iOS, but it should definitely be possible to compile Git LFS on such a version by hand if you can acquire the appropriate tools.  Git LFS works fine on ARM64 Darwin, which is what's also used in Apple Silicon machines, so I can't imagine any reason it shouldn't work for iOS.

I believe [Working Copy](https://workingcopy.app/) is an option, and while I've heard nice things about it, I can't speak to it myself.  There may be other options as well.
